modules/appeals_api/spec/models/appellant_spec.rb

```diff
@@ -90,6 +90,19 @@
     it { expect(claimant_appellant.zip_code_5).to eq '48070' }
   end
 
+  describe '#international_postal_code' do
+    let(:auth_headers) { fixture_as_json 'valid_200995_headers_extra.json', version: 'v2' }
+    let(:form_data) { (fixture_as_json 'valid_200995_extra.json', version: 'v2') }
+    let(:claimant_form_data) { form_data.dig('data', 'attributes', 'claimant') }
+
+    let(:claimant_appellant) do
+      described_class.new(auth_headers: auth_headers, form_data: claimant_form_data, type: :claimant)
+    end
+
+    it { expect(claimant_appellant.zip_code_5).to eq '00000' }
```

Comment:
tests to make sure that we accept 00000 for zipCode5 (since that's what is entered in the case of an international postal code, because we still have to send an 5-digit postal code to Central Mail, and we send zeros in this case)